src/components/Campaign.tsx

```diff
@@ -110,20 +117,24 @@ export const Campaign = () => {
                 ) : (
                   false
                 )}
-                <li>
-                  <button
-                    type="button"
-                    onClick={save}
-                    className="dp-button button-medium primary-green"
-                  >
-                    <FormattedMessage id="save" />
-                  </button>
-                </li>
               </ul>
             </EditorTopBar>
           </Header>
           <Footer>
-            <EditorBottomBar {...continuationUrls}></EditorBottomBar>
+            <EditorBottomBar {...continuationUrls}>
```

Comment:
`{...continuationUrls}` should be not necessary anymore

```suggestion
            <EditorBottomBar>
```

Reply:
That is fixed in this PR https://github.com/FromDoppler/doppler-editors-webapp/pull/431/files#diff-1b01b1e266a5f2916f9f38dc3a1d2488ebd7da80c9554a407a1f529734947d77